geomstats/geometry/embedded_manifold.py

```diff
@@ -28,6 +34,11 @@ def intrinsic_to_extrinsic_coords(self, point_intrinsic):
         ----------
         point_intrinsic : array-like, shape=[..., dim]
             Point in the embedded manifold in intrinsic coordinates.
+
+        Returns
+        -------
+        point_extrinsic : array-like, shape=[..., dim_embedding]
```

Comment:
what about `ambiant_dim`?

Reply:
We make the distinction between embedding manifold and ambient manifold. Ambient manifold is the manifold to which landmarks or curves belong in Landmarks and DiscreteCurves classes: it is not an embedding manifold.

Here, we really want the embedding manifold with its dimension.

Reply:
Ok